packer/build-windows-2016-vs-2017.json

```diff
@@ -92,7 +92,8 @@
         "choco install doxygen.install -confirm",
         "choco install openssl -confirm",
         "choco install strawberryperl -confirm",
-        "choco install nuget.commandline -confirm"
+        "choco install nuget.commandline -confirm",
+        "choco install dotnetcore-sdk -confirm"
```

Comment:
This is a great question.  I dunno what version the generic `choco install` of the SDK currently gives you (my money's on 3.1, or our stuff wouldn't build), but Chocolatey is flaky in general and I would prefer not to count on it always giving us the version we wish to use.

Reply:
Yes, it looks like that `dotnetcore-sdk` pulls the latest LTS. There are names for other versions but none that follow the 3.1 LTS specifically. I imagine that will have to come when the next LTS drops since they will overlap for a bit. I would keep my eyes on that to be safe. Optionally there are specific names for LTS updates to 3.1 that could be manually updated each time a new update level is released.

https://community.chocolatey.org/packages?q=dotnetcore-sdk

Reply:
We build against 3.1. Choco documentation says this package is 3.1. The bash install script specifies 3.1 explicitly.